src/main.ts

```diff
@@ -10,7 +10,6 @@ import router from '@/router'
 import { store } from '@/store'
 import Affix from 'vue-affix'
 import Vue2Filters from 'vue2-filters' // needed by SbcFeeSummary
-import { featureFlags, initLDClient } from '@/common/FeatureFlags'
 
```

Comment:
This was a recent fix that you're undoing. Please revert this change.

Reply:
poor rebase on my part, corrected in latest commit